src/server/handle-api.js

```diff
@@ -1,4 +1,4 @@
-import axios from 'axios'
+import fetch from 'isomorphic-fetch'
```

Comment:
Is it worth pulling out the server side `x-content-length` header bits? I did have a bit of a plan for spoofing on progress which might've made it a bit more accurate... Nevertheless... answering my own question, it's a bit of complexity for not much gain or accuracy. Let's pull out the `x-content-length` bits too.